Automattic/wp-calypso
# DIFM Lite: Handler for submitting website content form

#### Changes proposed in this Pull Request.

This will be merged into the parent PR: **https://github.com/Automattic/wp-calypso/pull/59895**

* Adds a `submitWebsiteContent` function which calls the `/sites/${ siteSlug }/do-it-for-me/website-content` API and saves form input to the backend.

#### Testing instructions



* Purchase DIFM Lite by proceeding through the flow at /start/do-it-for-me.
* After checkout, fill in the required fields on the Website Content step.
* Confirm that clicking on the last "Next" button first shows the processing screen and then takes you to `/home`.
* Additional test instructions: 2baae-pb/#plain
